packages/calcite-components/src/components/rating/rating.e2e.ts

```diff
@@ -766,4 +770,59 @@ describe("calcite-rating", () => {
       expect(element).toEqualAttribute("value", "2");
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-rating-spacing": {
+          selector: `calcite-rating`,
+          targetProp: "--calcite-internal-rating-spacing",
+        },
+        "--calcite-rating-outline-color": {
+          shadowSelector: `.${CSS.star}`,
+          targetProp: "color",
+        },
+        "--calcite-rating-outline-color-hover": {
+          shadowSelector: "label",
```

Comment:
This element has a class available and is used above, can you update this and similar ones to use `.${CSS.star}`?

Reply:
Yeah, I remember now.  I changed it to look for the `<label>` element because this `.hovered` selector didn't work:

<img width="723" alt="image" src="https://github.com/Esri/calcite-design-system/assets/821864/029ec005-167f-4437-82ac-d15ddb775874">

Given this test code:
```
"--calcite-rating-outline-color-hover": {
  shadowSelector: `.${CSS.hovered}`,
  targetProp: "color",
  state: "hover",
},
```

Reply:
Is using `CSS.star` ok (the hovered and selected classes get added to the same node)?  I got that to work for some reason, but using `.hovered` and `.selected` don't seem to work even though I can see them being added in the demo HTML.